includes/frontend/class-wcal-checkout-process.php

```diff
@@ -214,10 +216,11 @@ public function wcal_updated_recovered_cart( $cart_id, $order_id, $wcal_check_em
 
 				// Add Order Note.
 				$order->add_order_note( __( 'This order was abandoned & subsequently recovered.', 'woocommerce-abandoned-cart' ) );
-				delete_post_meta( $order_id, 'wcal_abandoned_cart_id' );
-				delete_post_meta( $order_id, 'wcal_recover_order_placed' );
-				delete_post_meta( $order_id, 'wcal_recover_order_placed_sent_id' );
-				delete_post_meta( $order_id, 'wcal_recovered_email_sent' );
+				$order->delete_meta_data( $order_id, 'wcal_abandoned_cart_id' );
```

Comment:
Line 219 to 222. Please remove `$order_id` as a parameter in the `delete_meta_data()`